baseapp/baseapp.go

```diff
@@ -98,6 +97,13 @@ type BaseApp struct { // nolint: maligned
 	// ResponseCommit.RetainHeight.
 	minRetainBlocks uint64
 
+	// application's version string
+	version string
+
+	// application's protocol version that increments on every upgrade
+	// if BaseApp is passed to the upgrade keeper's NewKeeper method.
+	appVersion uint64
```

Comment:
[not blocking] can these two include an example to help further differentiate them?

If `version = 2.0.0` then is `appVersion = 2`? Is appVersion always expected to be the major version of the version string?